pyspark_ai/pyspark_ai.py

```diff
@@ -480,6 +481,7 @@ def _get_sql_query(
             )
         else:
             # Otherwise, generate the SQL query with a prompt with few-shot examples
+            print(f"-------------------------Start generating sql query with a prompt with few-shot examples-------------------------\n\n")
```

Comment:
@PZD-CHINA maybe we can define a method for log output like this? That will make code cleaner.